library/include/hipsolver.h

```diff
@@ -239,6 +239,63 @@ HIPSOLVER_EXPORT hipsolverStatus_t hipsolverZgetrf(hipsolverHandle_t       handl
                                                    int*                    devIpiv,
                                                    int*                    devInfo);
 
+// potrf
+HIPSOLVER_EXPORT hipsolverStatus_t hipsolverSpotrf_bufferSize(
+    hipsolverHandle_t handle, hipsolverFillMode_t uplo, int n, float* A, int lda, int* lwork);
+
+HIPSOLVER_EXPORT hipsolverStatus_t hipsolverDpotrf_bufferSize(
+    hipsolverHandle_t handle, hipsolverFillMode_t uplo, int n, double* A, int lda, int* lwork);
+
+HIPSOLVER_EXPORT hipsolverStatus_t hipsolverCpotrf_bufferSize(hipsolverHandle_t   handle,
+                                                              hipsolverFillMode_t uplo,
+                                                              int                 n,
+                                                              hipsolverComplex*   A,
```

Comment:
I mentioned it elsewhere, but maybe we should just skip over defining a separate hipsolverComplex type and just use `hipFloatComplex` and `hipDoubleComplex` directly. Or at least make our `hipsolverComplex` types as typedefs for the corresponding HIP types.

Reply:
I agree. I'll open a separate PR to switch over to the hip complex types, probably after all the functions are in.